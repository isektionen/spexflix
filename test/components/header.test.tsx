import React from 'react'
import { render } from '../testUtils'
import Header from '../../components/header'

describe('Footer component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Header title="foo" />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
