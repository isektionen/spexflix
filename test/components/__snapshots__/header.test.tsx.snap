// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    <a
      href="/"
    >
      <div
        class="logotype"
      >
        <div
          class="letter"
        >
          <span
            style="transform: rotateY(10deg) scaleY(1.01);"
          >
            F
          </span>
        </div>
        <div
          class="allButFirstLetter"
        >
          <div
            class="letter"
          >
            <span
              style="transform: rotateY(0deg) scaleY(1);"
            >
              O
            </span>
          </div>
          <div
            class="letter"
          >
            <span
              style="transform: rotateY(-10deg) scaleY(1.01);"
            >
              O
            </span>
          </div>
        </div>
      </div>
    </a>
    <nav
      class="navigation"
    >
      <ul
        class="navList"
      >
        <li
          class="navItem"
        >
          <a
            class="navLinkActive"
            href="/"
          >
            Home
          </a>
        </li>
      </ul>
    </nav>
  </header>
</DocumentFragment>
`;
